units/commit_queue.go

```diff
@@ -161,6 +186,16 @@ func (j *commitQueueJob) Run(ctx context.Context) {
 	v, err := makeVersion(ctx, githubToken, projectRef.Identifier, pr)
 	if err != nil {
 		j.AddError(errors.Wrap(err, "can't make version"))
+		j.AddError(sendCommitQueueGithubStatus(j.env, pr, message.GithubStateFailure, "can't make version", ""))
```

Comment:
At this point in the code we have enough information about the PR to send a status to GitHub indicating the failure. This is important so the user isn't left wondering why "evergreen merge" isn't working.